### DISCUSSION TITLE
How to set up new user with limited access to certain queues?

### DISCUSSION THREAD

**Discussion by jbro971** [arkmq-org/arkmq-org-broker-operator#996] [Q&A] [answered]
Hello, do you have a working example on how to set up new user, new role and limited access (lets say SEND, CONSUME) to queues like xy and xyz? Thank you very much

**Comment by gtully** [comment, marked as answer]
There are some examples in the tests, peek at: https://github.com/artemiscloud/activemq-artemis-operator/blob/main/controllers/activemqartemissecurity_broker_properties_test.go#L113

user management through a jaas secret, that could delegate to any login module. Then using properties to configure the broker rbac.
